src/dependency-manager/utils/parser.ts

```
@@ -197,6 +196,12 @@ export class ArchitectParser {
             value = node.arguments[0].value.trim();
           } else if (node.callee.value === 'startsWith') {
             value = node.arguments[0].value.startsWith(node.arguments[1].value);
+          } else if (node.callee.value === 'parseUrl') {
+            try {
+              value = (new URL(node.arguments[0].value) as any)[node.arguments[1].value as string];
+            } catch {
+              value = '';
```

Should this silently error?